css/_atlaskit_overrides.scss

```diff
@@ -103,3 +103,20 @@ div.Tooltip {
     line-height: 14px;
     padding: 8px;
 }
+
+// make modal full screen on landscape orientation
+@media (max-height: 420px) {
+    .atlaskit-portal {
+        .css-1oc7v0j {
```

Comment:
is this a static class name? I think using what to appears to be a generated class name is not safe.

Reply:
This is a generated class name. But as long as the library is not updated the name will be preserved (the class name will be generated with the same name). There is no other way to select that element given the dom structure. Also I've seen the use of such classes in the project.

Reply:
Is there is a selector we can use instead of the name great, but if not, oh well, we've been in there before...